### PR TITLE
Colon in Dashboard Name Search

### LINKED ISSUE #6282: Suggestion: Changing the language on search results for dashboards
A very minor suggestion, but can we perhaps change the language used for the suggested results around dashboards in PostHog's search feature? 

Currently the language we use is 'Go to [Feature Name] [Item Name]'. The only item names which appear to come up in search suggestions are around dashboards, but this can create results which are hard to parse, eg.:

_Go to Dashboard Tiina Test
Go to Dashboard Default
Go to Dashboard 🚀 A Product Dashboard_

It can also lead to results which could be confusing and indicate a different result, depending on the name of the dashboard, eg:

_Go to Dashboard Plugins_ (Could suggest either a dashboard about plugins, or plugins that work with the dashboard)
_Go to Dashboard Default
Go to Dashboard Logins
Go to Dashboard Content_

Icons are used to mitigate some of these issues, but not all. 

A simple solution would be to change the language to 'Go to [Feature Name]: [Item Name]', with the colon only appearing if an item name is shown. This would me existing terms remain as they are (e.g. _'Go to Persons'_), but dashboards become clearer (e.g. _'Go to Dashboard: Plugins', 'Go to Dashboard: Tiina Test'_). 

Screenshot 2021-10-06 at 13 57 07

### PR DESCRIPTION
**Pull request by marcushyett-ph**
## Changes

Closes: #6282

Adds a colon to make parsing a dashboard name in search clearer

## How did you test this code?

Tried it out with my local dashboard
image

### CODE REVIEW

**Review by joethreepwood — APPROVED** [review]

**Comment by mariusandra** [comment]
Feel free to merge once approved!
